src/client/components/body-portal/body-portal.tsx

```diff
@@ -58,14 +65,17 @@ export class BodyPortal extends React.Component<BodyPortalProps, {}> {
       if (BodyPortal.aboveAll) throw new Error("There can be only one");
       BodyPortal.aboveAll = this;
     }
+
+    this.setState({ isMounted: true });
   }
   componentWillUnmount() {
     document.body.removeChild(this.target);
     if (BodyPortal.aboveAll === this) BodyPortal.aboveAll = undefined;
   }
 
   render() {
+    const { isMounted } = this.state;
     Object.assign(this.target.style, normalizeStyles(this.props));
-    return ReactDOM.createPortal(this.props.children, this.target);
+    return ReactDOM.createPortal(isMounted && this.props.children, this.target);
```

Comment:
Is calling `createPortal` with `false` safe?

Reply:
I believe so. It has no effect in the running app (i.e. no warnings printed to console) and it's typed as a `ReactNode` which is a union including `boolean` (`type ReactNode = ReactChild | ReactFragment | ReactPortal | string | number | boolean | null | undefined;`).

The React API documentation says basically nothing for `createPortal()`, but [`React.Component`'s `render()`](https://reactjs.org/docs/react-component.html#render) method allows returning a boolean specifically to support the `test && <Child />` pattern.